src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -725,6 +727,11 @@ static QString slurTieLineStyle(const SlurTie* s)
     if (!lineType.isEmpty()) {
         rest = QString(" line-type=\"%1\"").arg(lineType);
     }
+    if (s->slurDirection() != engraving::DirectionV::AUTO) {
+        rest += QString(" type=\"start\" placement=\"%1\"")
```

Comment:
That first part, `" type=\"start\""`needs to be written in any case, only the `placement`part is dependant of `slurDirection`. And the unit tests need to get adjusted for that.